### DISCUSSION TITLE
Public Bot, Guild Install, How to config now?

### DISCUSSION THREAD

**Discussion by Lokeser** [jagrosh/MusicBot#1657] [Questions]
So, now discord has a new type of bot install, user install and guild install. I have to turn it on guild install to make the bot work and invite to server, but he cant be public to work. Im trying to solve this, but i may be dumb, could someone helps?

**Comment by ghost** [comment]
disable the install links (both guild and user) and then you should be able to disable public bot, and use the old oauth to invite the bot to the server.
